src/entities/Event.ts

```diff
@@ -0,0 +1,78 @@
+import { Entity, PrimaryGeneratedColumn, Column, ManyToMany, JoinTable, OneToMany } from 'typeorm';
+import { AppUser, Attendance, RSVP } from '@Entities';
+
+export enum EventType {
+  PROFESSIONAL = 'professional',
+  SOCIAL = 'social',
+  TECHNICAL = 'technical',
+  MENTORSHIP = 'mentorship',
+}
+
+export enum EventStatus {
+  PENDING = 'pending',
+  READY = 'ready',
+  COMPLETE = 'complete',
+}
+
+@Entity()
+export class Event {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column('text')
+  description: string;
+
+  @Column({ nullable: true })
+  location: string;
+
+  @Column('timestamp')
+  startDate: string;
+
+  @Column('timestamp')
+  endDate: string;
+
+  @Column({
+    type: 'enum',
+    enum: EventType,
+    nullable: true,
+  })
+  type: string;
+
+  @Column({
+    type: 'enum',
+    enum: EventStatus,
+    default: EventStatus.PENDING,
+  })
+  status: string;
+
+  @ManyToMany(() => AppUser, { eager: true })
+  @JoinTable()
+  hosts: AppUser[];
+
+  @OneToMany(
+    () => Attendance,
+    attendance => attendance.event
```

Comment:
Just curious on what this line means

Reply:
https://github.com/typeorm/typeorm/blob/master/docs/many-to-one-one-to-many-relations.md